src/mmw/js/src/modeling/views.js

```diff
@@ -66,6 +67,17 @@ var ModelingHeaderView = Marionette.LayoutView.extend({
 
     onShow: function() {
         this.reRender();
+    },
+
+    saveAfterLogin: function(user, guest) {
+        if (!guest && this.model.isNew()) {
+            var user_id = user.get('id');
+            this.model.set('user_id', user_id);
```

Comment:
The POST there in https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/views.py#L47 will be for creating Projects, and since the view is decorated with an `Authenticated` class, it will always be created in that user's context.

For project updates, this bit of code on the backend is preventing cross-user updates:
https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/apps/modeling/views.py#L68

I believe there is some pretty good test coverage on this functionality, too.